alpine-ruby/Dockerfile

```diff
@@ -29,9 +29,9 @@ RUN echo 'gem: --no-document' > ~/.gemrc \
 
 # Setup bundler for the root user
 RUN bundle config --global -- build.nokogiri '--use-system-libraries' \
-  && bundle config --global -- frozen 'true' \
+  && bundle config --global -- frozen '1' \
```

Comment:
Yeah, this is likely the better tested code path. The fact they support both makes me worry so we should stick to the more likely not to break one.